internal/driver/driver.go

```diff
@@ -78,14 +88,22 @@ func (d *Driver) Initialize(lc logger.LoggingClient, asyncCh chan<- *sdkModel.As
 	return nil
 }
 
+var (
+	GetEndpoints = opcua.GetEndpoints
+)
+
+var (
+	SelectEndPoint = opcua.SelectEndpoint
+)
+
 // creates the options to connect with a opcua Client based on the configured options.
 func (d *Driver) createClientOptions() ([]opcua.Option, error) {
-	availableServerEndpoints, err := opcua.GetEndpoints(d.serviceConfig.OPCUAServer.Endpoint)
```

Comment:
here I would let it be the same as it was before opcua.GetEndpoints, this makes it clear and readable as compared to the newer version IMHO as we don't know if it is something being called out of the object or something and for clarification we have to look above.

Reply:
This was done for mocking, by doing this I can substitute GetEndpoints with a mock in unit tests if I want to test createClientoptions. I added a comment to make it clearer